test/GogeDao.t.sol

```diff
@@ -357,8 +1109,7 @@ contract DaoTest is Utility, Test {
 
         // Verify poll metadata
         assertEq(gogeDao.getMetadata(1).description, "I want to propose a tax change");
-        assertEq(gogeDao.getMetadata(1).time1, block.timestamp);
-        assertEq(gogeDao.getMetadata(1).time2, block.timestamp + 2 days);
+        assertEq(gogeDao.getMetadata(1).endTime, block.timestamp + 2 days);
```

Comment:
Even in the test cases we're only ever using and comparing to the `Metadata` struct, which should be how it is in the contract!